lxqt-config-input/CMakeLists.txt

```diff
@@ -61,8 +68,10 @@ target_link_libraries(lxqt-config-input
     Qt5::Widgets
     Qt5::X11Extras
     ${X11_LIBRARIES}
+    ${X11_Xinput_LIB}
```

Comment:
IMO, it's better to use the `*_LDLIBS`

Reply:
X11 appears to be a special case, where there's only `*_LIB`, but no `*_LDLIBS` or `*_LDFLAGS`. The latter two does not build on my machine (Arch Linux).

```
X11_Xinput_INCLUDE_PATH,       X11_Xinput_LIB,     X11_Xinput_FOUND
```

(From https://cmake.org/cmake/help/v3.13/module/FindX11.html)

Reply:
OK. Sorry for the noise :(